command/genesis/polybft_params.go

```diff
@@ -0,0 +1,224 @@
+package genesis
+
+import (
+	"encoding/hex"
+	"fmt"
+	"path"
+	"strings"
+	"time"
+
+	"github.com/0xPolygon/polygon-edge/chain"
+	"github.com/0xPolygon/polygon-edge/command"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft/bitmap"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft/polybftcontracts"
+	"github.com/0xPolygon/polygon-edge/server"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/umbracle/ethgo"
+)
+
+const (
+	validatorPrefixPathFlag = "prefix"
+
+	validatorSetSizeFlag = "validator-set-size"
+	sprintSizeFlag       = "sprint-size"
+	blockTimeFlag        = "block-time"
+	validatorsFlag       = "polybft-validators"
+
+	defaultEpochSize        = uint64(10)
+	defaultSprintSize       = uint64(5)
+	defaultValidatorSetSize = 100
+	defaultBlockTime        = 2 * time.Second
+
+	bootnodePortStart = 30301
+)
+
+var (
+	// hard code address for the sidechain (this is only being used for testing)
+	ValidatorSetAddr         = ethgo.HexToAddress("0xBd770416a3345F91E4B34576cb804a576fa48EB1")
+	SidechainBridgeAddr      = ethgo.HexToAddress("0x5a443704dd4B594B382c22a083e2BD3090A6feF3")
+	sidechainERC20Addr       = ethgo.HexToAddress("0x47e9Fbef8C83A1714F1951F142132E6e90F5fa5D")
+	SidechainERC20BridgeAddr = ethgo.HexToAddress("0x8Be503bcdEd90ED42Eff31f56199399B2b0154CA")
+)
+
+func (p *genesisParams) getPolyBftConfig(validators []*polybft.Validator) (*polybft.PolyBFTConfig, error) {
+	smartContracts, err := deployContracts()
+	if err != nil {
+		return nil, err
+	}
+
+	config := &polybft.PolyBFTConfig{
+		// TODO: Bridge
+		InitialValidatorSet: validators,
+		BlockTime:           p.blockTime,
+		EpochSize:           p.epochSize,
+		SprintSize:          p.sprintSize,
+		ValidatorSetSize:    p.validatorSetSize,
+		ValidatorSetAddr:    types.Address(ValidatorSetAddr),
+		SidechainBridgeAddr: types.Address(SidechainBridgeAddr),
+		SmartContracts:      smartContracts,
+	}
+
+	return config, nil
+}
+
+func (p *genesisParams) GetChainConfig() (*chain.Chain, error) {
```

Comment:
You are generating a new genesis configuration here, so the name should probably be `GeneratePolyBFTConfig`

Reply:
done

Reply:
Fixed where?

Reply:
Fixed 00883a114655574ae6dfe3160ed2f888b17449ae